### ISSUE TITLE
Can I draw multiple curves in a single figure ?

### ISSUE THREAD

**Issue by sunshineatnoon** [torrvision/crayon#25]
I use `foo.add_scalar_dict({'train_acc':trainacc,'test_acc':testacc})` to send training and testing accuracy to the server, but they ended up in separate figures. How can I draw them in a single figure ?

**Comment by albanD** [comment]
Doesn't using the `split on underscores` option in the tensorboard UI solves this problem?

**Comment by sunshineatnoon** [comment]
@albanD Thanks for your reply. The `split on underscores` shows two accuracies side by side like this. 

![](https://raw.githubusercontent.com/sunshineatnoon/Paper-Collection/master/%5B%22images%22%5D/tensorboard.png)

But I would like to show these two accuracies in a single figure so that I can see how overfit the model is. Just like [torch's logger](https://github.com/torch/optim/blob/master/doc/logger.md) .

**Comment by albanD** [comment]
The only way I know to trick tensorboard into doing that is to create two experiments with different names (train and test in your case) and then log a scalar with the exact same name in both (acc in your case).

**Comment by sunshineatnoon** [comment]
@albanD Thanks, I will try that. BTW, Does pytorch has plan to implement [logger](https://github.com/torch/optim/blob/master/doc/logger.md) or [display](https://github.com/szym/display) module like Torch?

**Comment by albanD** [comment]
The torch tnt framework is planned to be moved to pytorch here: https://github.com/pytorch/tnt
